src/core/styles/fonts.css

```
@@ -0,0 +1 @@
+@import url('https://googleapis.com');
```

Stylelint (17.9.0)

[error] Expected "url('https://googleapis.com')" to be "'https://googleapis.com'" (import-notation)

(import-notation)